### PR TITLE
Update ci configuration and fixed the code error and security issue

### LINKED ISSUE #386: [Security issue] for all the projects are imported mesh kit
**Description**

All the components of meshery are used meskit. And all the adapters have the issues of security like https://github.com/meshery/meshkit/security/dependabot/59.

And in one of adapters like https://github.com/meshery/meshery-consul/security/dependabot

More detail please check here.
https://github.com/meshery/meshkit/security/dependabot

**Expected Behavior**

meshkit should release a newer version with all these were fixed.

**Screenshots**

**Environment:**
 - OS: [e.g. Ubuntu]
 - Browser: [e.g. Chrome, Safari]
 - Version: [e.g. 22]
 - Device: [e.g. laptop, iPhone 8]

---
[Optional] **To Reproduce**

[Optional] **Additional Context**

### PR DESCRIPTION
**Pull request by Aisuko**
**Description**

This PR fixes #386 

**Notes for Reviewers**
This PR fixed:
https://github.com/meshery/meshkit/security/dependabot/57
https://github.com/meshery/meshkit/security/dependabot/58
https://github.com/meshery/meshkit/security/dependabot/59
https://github.com/meshery/meshkit/security/dependabot/63

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: cmd/errorutil/internal/coder/commands.go [hunk 1]
```diff
@@ -123,15 +123,15 @@ func commandUpdate() *cobra.Command {
...
 			if err != nil {
 				return err
 			}
-			updateAll, err := cmd.Flags().GetBool(forceUpdateAllCodesCmdFlag)
+			updateAll, err = cmd.Flags().GetBool(forceUpdateAllCodesCmdFlag)
 			if err != nil {
 				return err
 			}
-			return walkSummarizeExport(globalFlags, true, updateAll)
+			return walkSummarizeExport(gFlags, true, updateAll)
```
Comment (leecalcote) [inline cmd/errorutil/internal/coder/commands.go:134]: @zakisk look good?

File: go.mod [hunk 2]
```diff
@@ -17,9 +17,9 @@ require (
 	github.com/docker/compose/v2 v2.2.0
 	github.com/go-git/go-git/v5 v5.4.2
 	github.com/go-logr/logr v1.2.4
-	github.com/google/uuid v1.3.0
+	github.com/google/uuid v1.3.1
 	github.com/kubernetes/kompose v1.26.1
-	github.com/layer5io/meshery-operator v0.6.10
+	github.com/layer5io/meshery-operator v0.6.13
```
Comment (leecalcote) [inline go.mod:22]: 👏

File: utils/manifests/generateComponent.go [hunk 3]
```diff
@@ -19,7 +19,7 @@ func GenerateComponents(ctx context.Context, manifest string, resource int, cfg
 	crds = cfg.ExtractCrds(manifest)
 	for _, crd := range crds {
 		var parsedCrd cue.Value
-		if cfg.CrdFilter.IsJson != true {
+		if !cfg.CrdFilter.IsJson {
```
Comment (leecalcote) [inline utils/manifests/generateComponent.go:22]: Good

File: utils/manifests/utils.go [hunk 4]
```diff
@@ -156,70 +122,6 @@ func RemoveHelmTemplatingFromCRD(crdyaml *string) {
...
-// cleanup
-func deleteFile(path string) error {
-	err := os.Remove(path)
-	if err != nil {
-		return err
-	}
-	return nil
-}
-
-//TODO: After finalizing on where to keep dictionary.json, use that file to initialize dict map in init()
```
Comment (leecalcote) [inline utils/manifests/utils.go:204]: I think we still have this as a TODO. If we're deleting it as a comment, do you mind opening up an issue with the same information in order to not lose track of this to-do item?

File: utils/manifests/utils.go [hunk 5]
```diff
@@ -156,70 +122,6 @@ func RemoveHelmTemplatingFromCRD(crdyaml *string) {
 	*crdyaml = strings.Join(yamlArr, "\n---\n")
 }
 
-func getCrdnames(s string) []string {
```
Comment (leecalcote) [inline utils/manifests/utils.go:159]: @MUzairS15, will you confirm that Meshery Server/other doesn't use this function or the filterYaml() function any longer?

File: utils/manifests/utils.go (hunk 4 above)
Comment (Aisuko) [reply utils/manifests/utils.go:204]: #391

Comment (leecalcote) [reply utils/manifests/utils.go:204]: Perfectó

File: utils/manifests/utils.go (hunk 5 above)
Comment (leecalcote) [reply utils/manifests/utils.go:159]: @KiptoonKipkurui you might be able to offer similar confirmation

Comment (leecalcote) [reply utils/manifests/utils.go:159]: Confirmed: No use inside of Meshery Server.

Comment (Aisuko) [reply utils/manifests/utils.go:159]: Good check

**Pull request merged**